sklearn/manifold/_t_sne.py

```diff
@@ -753,7 +748,7 @@ def __init__(
         method="barnes_hut",
         angle=0.5,
         n_jobs=None,
-        square_distances="legacy",
+        square_distances="warn",
```

Comment:
I would not call the default "warn", because the code only actually warns when the value is not the default. Maybe "deprecated" ?

Reply:
True